libs/core/src/lib/avatar/avatar.component.ts

```diff
@@ -40,7 +42,8 @@ const ALTER_ICON_OPTIONS = {
     changeDetection: ChangeDetectionStrategy.OnPush,
     host: {
         '[attr.tabindex]': '_tabindex'
-    }
+    },
+    providers: [DestroyedService, skeletonConsumer()]
```

Comment:
that's example how we add skeleton state for the simple components (one placeholder).